server/src/models/index.js

```diff
@@ -0,0 +1,13 @@
+// const dbConfig = require('../.env.example');
```

Comment:
I am a little bit confused what that file is doing exactly. Lets discuss this today in the meeting.

Reply:
Removed, was left from another implementation version